EIPS/eip-4950.md

```diff
@@ -0,0 +1,90 @@
+---
+eip: 4950
+title: Entangled Tokens
+description: ERC-721 extension with two tokens minted that are tied together
+author: Victor Munoz (@victormunoz)
+discussions-to: https://ethereum-magicians.org/t/entangled-tokens/8702
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-28
+requires: 721, 1155
```

Comment:
```suggestion
requires: 20, 721, 1155
```

Since this references `EIP-20`, notably in the Spec, `20` must be added to the `requires` list.